test/test_torch.py

```diff
@@ -16341,6 +16344,7 @@ def _test_addmm_addmv(self, f, t, m, v, *, alpha=None, beta=None, transpose_out=
     @dtypesIfCUDA(torch.half, torch.float, torch.double, torch.cfloat, torch.cdouble)
     @dtypes(torch.bfloat16, torch.float, torch.double, torch.cfloat, torch.cdouble)
     @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    @tf32_on_and_off(0.05)
```

Comment:
hm, I'm surprised it's needed here, why would addmv ever use tf32?

Reply:
Shouldn't addmv use tensor cores? It is equivalent to a MxN @ Nx1 `addmm`

Reply:
addmv is bandwidth-bound, so it does not benefit from tensor cores. I don't know how cublas is set up internally, so who knows if it is using different math or not.

Reply:
Tested that `addmv` on TF32 is not faster than FP32. So I disabled TF32 on `addmv`, so there is no percision loss.